src/themes/teams/components/Menu/menuStyles.ts

```diff
@@ -28,7 +28,7 @@ export default {
             ...solidBorder(variables.primaryBorderColor),
           }),
           borderRadius: pxToRem(4),
-          overflow: 'hidden',
+          // overflow: 'hidden',
```

Comment:
this needs to be verified - it breaks submenu when oveflow is hidden

Reply:
This is the PR that added it: https://github.com/stardust-ui/react/pull/360 Will sync with @Bugaa92 for finding alternative solution for the cropped borders, I have new ideas.